tests/database/odbc-w-enclave/README.md

```diff
@@ -0,0 +1,39 @@
+# Database Test Case (ODBC Attestation)
+
+## Description
+
+This is the test case for database connection and query from sgxlkl application to a remote SQLServer database with always encrypted V2. After connection with attestation established, the test will do following queries and operations:
+
+1. before start, drop the test data table, CEK(Column Encrypted Key), CMK(Column Master Key)
```

Comment:
Since this test may run concurrently in different configurations in CI (debug, non-release, etc), does this actually work? Should the test table name be generated randomly?

Reply:
That is a good point I have not considered when writing the test case. But that will also raise other concerns. This test case will also include CEK(Column Encrypted Key), CMK(Column Master Key) creation and drop. Will that make the test case too complicated? What is your opinion @jxyang @Pengpeng-Microsoft

Reply:
I would recommend a postfix to CEK/CMK/table by combination of configuration and random number. Beyond that, you can also do a lock record based checking mechanism to be more cautious.

Reply:
Using a suffix(timestamp+random number) for all database objects. A registry table has also been created in the database, for every table/cmk/cek creation, a record will be registered in registry. At the end of the test, table, cmk and cek will be dropped. For each successful drop operation, related registry record will be deleted.